python/paddle/fluid/tests/unittests/test_reduce_op.py

```diff
@@ -365,16 +367,20 @@ def test_check_output(self):
         self.check_output(check_eager=True)
 
     def test_check_grad(self):
-        self.check_grad(['X'], 'Out', check_eager=True)
+        self.check_grad(['X'], 'Out', check_eager=True, check_prim=True)
 
 
 class TestProdOp_ZeroDim(OpTest):
     def setUp(self):
         self.python_api = paddle.prod
         self.op_type = "reduce_prod"
+        self.prim_op_type = "prim"
         self.inputs = {'X': np.random.random([]).astype("float64")}
         self.outputs = {'Out': self.inputs['X'].prod()}
         self.attrs = {'dim': [], 'reduce_all': True}
+        # reduce doesn't support float64 in cinn.
```

Comment:
will it raise error when test fp64 with other test(not 0D tensor) in cinn?

Reply:
fp64 added in test